boards/nordic/nrf54l15dk/doc/index.rst

```diff
@@ -98,7 +98,7 @@ to be build as multicore configuration with code snippet called ``vpr_launcher``
 for the application core.
 
 Enter the following command to compile ``hello_world`` for the FLPR core::
- west build -p -b nrf54l15pdk/nrf54l15/cpuflpr --sysbuild -- -DSB_VPR_LAUNCHER=y
+ west build -p -b nrf54l15dk/nrf54l15/cpuflpr --sysbuild -- -DSB_VPR_LAUNCHER=y
```

Comment:
This should be corrected upstream, not in this noup commit.

Reply:
True.